espresso/core/javatests/androidx/test/espresso/EspressoTest.java

```diff
@@ -101,20 +113,44 @@ public void openOverflowFromActionBar() {
     assertThat(
         tracer.getSpans(),
         contains(
-            "beginSpan: Espresso-perform-AdapterDataLoaderAction",
-            "+-endSpan: Espresso-perform-AdapterDataLoaderAction",
-            "beginSpan: Espresso-perform-GeneralClickAction",
-            "+-endSpan: Espresso-perform-GeneralClickAction",
-            "beginSpan: Espresso-perform-GeneralClickAction",
-            "+-endSpan: Espresso-perform-GeneralClickAction",
-            "beginSpan: Espresso-openActionBarOverflowOrOptionsMenu",
-            "beginSpan: Espresso-perform-KeyEventAction",
-            "+-endSpan: Espresso-perform-KeyEventAction",
-            "+-endSpan: Espresso-openActionBarOverflowOrOptionsMenu",
-            "beginSpan: Espresso-perform-GeneralClickAction",
-            "+-endSpan: Espresso-perform-GeneralClickAction",
-            "beginSpan: Espresso-check-MatchesViewAssertion",
-            "+-endSpan: Espresso-check-MatchesViewAssertion"));
+            "beginSpan: Espresso.onView(is assignable from class <class"
+                + " android.widget.AdapterView>)",
+            "+-endSpan: Espresso.onView(is assignable from class <class"
+                + " android.widget.AdapterView>)",
+            "beginSpan: Espresso.perform.AdapterDataLoaderAction(is assignable from class <class"
+                + " android.widget.AdapterView>",
+            "+-endSpan: Espresso.perform.AdapterDataLoaderAction(is assignable from class <class"
+                + " android.widget.AdapterView>",
+            "beginSpan: Espresso.onView(displaying data matching: (an instance of java.util.Map and"
```

Comment:
DX improvement: Could you make sure the ) is symmetric and the truncation is signaled with ... or similar?